realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -689,6 +689,32 @@ const rmw_qos_profile_t BaseRealSenseNode::qos_string_to_qos(std::string str)
 #endif
     if (str == "SYSTEM_DEFAULT")
         return rmw_qos_profile_system_default;
+    if (str == "POINTCLOUD_DEFAULT")
```

Comment:
pointcloud and info can use the same QoS : DEFAULT. All they change is the depth, from the original 10 to 1. I prefer, as much as we can, not to add new QoS options.
IMU can use HID_DEFAULT.

Reply:
I agree that too much QoS profiles can be confusing and not useful. The reason I declared distinct default QoS was mainly to ensure backward compatibility, and also to keep the different kind of publishers independent. If someday the default value of depth for the info topics need to be changed, it shouldn't be impacting the pointcloud publisher's depth, for instance. 

But I guess it can be managed when this will be needed (if it is needed one day), and I agree it may better to keep it simple with this PR.

I did the changes you asked for [here for the info and pointcloud publishers](https://github.com/IntelRealSense/realsense-ros/pull/1833/commits/5b344b9a88445efecf12b5e6d9268f0d3ed19536) and [here for the IMU publisher](https://github.com/IntelRealSense/realsense-ros/pull/1833/commits/773fdafbd58fcdbf599a416ed5e028fd681eb9bb). My main concern is still about the backward compatibility of the publishers' depths but you seemed to say it shouldn't be a problem to change the default depth. Some parameters could be opened for the user to set the publishers' depths. But again, too much settings could be confusing. Could you just confirm it is good for you this way?